src/batched/KokkosBatched_Util.hpp

```diff
@@ -303,6 +303,60 @@ struct Mode {
   };
 };
 
+#if !defined(KOKKOS_IF_HOST)
+
+template <class>
+struct algo_level3_blocked_mb_impl;
+template <>
+struct algo_level3_blocked_mb_impl<Kokkos::HostSpace> {
+  static constexpr int value = 4;
+};
+#if defined(KOKKOS_ENABLE_CUDA)
+template <>
+struct algo_level3_blocked_mb_impl<Kokkos::CudaSpace> {
+  static constexpr int value = 1;
```

Comment:
I messed that up in the refactor.  This should have been `2`.  This is probably what triggered the link time issue and then the unit test failure.  Testing it locally, will push force a fix shortly.